backend/serve.go

```diff
@@ -31,5 +40,27 @@ func Serve(opts ServeOpts) error {
 		pluginOpts.TransformServer = newTransformSDKAdapter(opts.TransformDataHandler)
 	}
 
+	grpc_prometheus.EnableHandlingTimeHistogram()
+	recoveryOption := grpc_recovery.WithRecoveryHandlerContext(recoveryHandler)
+
+	pluginOpts.GRPCServer = func(opts []grpc.ServerOption) *grpc.Server {
+		mergedOpts := []grpc.ServerOption{}
+		mergedOpts = append(mergedOpts, opts...)
+		sopts := []grpc.ServerOption{
+			grpc.MaxRecvMsgSize(maxServerReceiveMsgSize),
+			grpc.MaxSendMsgSize(maxServerSendMsgSize),
```

Comment:
Instead of constants, `maxServerReceiveMsgSize ` and `maxServerReceiveMsgSize` could be new fields in `ServeOpts` and set only if they are provided?

Reply:
Talked to Brian and he explained that reverting back to 4mb was due to that changes was needed on the Grafana side as well so the changes to 16mb didn't have effect. Will keep the default and let plugin override if needed.